smriprep/cli/run.py

```diff
@@ -95,6 +95,9 @@ def get_parser():
     g_ants.add_argument('--skull-strip-fixed-seed', action='store_true',
                         help='do not use a random seed for skull-stripping - will ensure '
                              'run-to-run replicability when used with --omp-nthreads 1')
+    g_ants.add_argument(
+        '--skip-brain-extraction', required=False, action='store_true',
+        default=False, help='Skip the brain extraction workflow (for pre-extracted T1ws))')
```

Comment:
Hmm. This takes care of false negatives, but I suppose we should consider false positives, where we might incorrectly decide an image is skull-stripped.

Maybe we want `--skull-strip {auto,skip,force}`?

Or am I over-thinking the problem?

Reply:
Yeah, I previously suggested this, but also worried I was overthinking it. False positives are actually probably a bigger problem in a way (many more people have normal T1s)

If you're comfortable with the  additional CLI complexity, I think it this makes the most sense.

Again, we could also remove the detection if we think its not reliable, and only have a skip detection flag.

My preferences in order: 1) CLI argument w/ 3 options 2) skip only CLI flag, with no detection 3) skip flag + detection (current)

Reply:
I lean toward 1 as well.

Reply:
@mgxd ? any opinion

Reply:
i'm fine with the 3 way approach, with the default being `auto`. It should definitely document which route was taken

Reply:
Sure, how? You mean which option was select or what auto detected? And if so, where do I log that?